macros/PGmatrixmacros.pl

```diff
@@ -481,13 +481,13 @@ sub dm_tth_delimeter {
         my ($top, $mid, $bot, $extra);
         my ($j, $out);
 
-        if($char eq "(") { ($top, $mid, $bot, $extra) = ('�','�','�','�');}
-        elsif($char eq ")") { ($top, $mid, $bot, $extra) = ('�','�','�','�');}
-        elsif($char eq "|") { ($top, $mid, $bot, $extra) = ('�','�','�','�');}
-        elsif($char eq "[") { ($top, $mid, $bot, $extra) = ('�','�','�','�');}
-        elsif($char eq "]") { ($top, $mid, $bot, $extra) = ('�','�','�','�');}
-        elsif($char eq "{") { ($top, $mid, $bot, $extra) = ('�','�','�','�');}
-        elsif($char eq "}") { ($top, $mid, $bot, $extra) = ('�','�','�','�');}
+        if($char eq "(") { ($top, $mid, $bot, $extra) = ('�','�','�','�');}
```

Comment:
Peter, it looks like your text editor didn't encode these accented characters correctly.